src/libraries/Common/src/System/Obsoletions.cs

```diff
@@ -48,5 +48,8 @@ internal static class Obsoletions
 
         internal const string WebRequestMessage = "Use HttpClient instead.";
         internal const string WebRequestDiagId = "SYSLIB0014";
+
+        internal const string DisablePrivateReflectionAttributeMessage = "DisablePrivateReflectionAttribute has no effect in .NET Core and .NET 5.0+ applications.";
```

Comment:
This attribute still has effect in a few corner cases in CoreCLR. Search for `DisablePrivateReflection` under `src\coreclr` and delete everything found.

Reply:
Also, we may want to say `.NET 6.0+` in the error message.